tests/test_classification.py

```diff
@@ -1,22 +1,24 @@
 from conftest import get_distances, get_means, get_metrics
 import numpy as np
 from numpy.testing import assert_array_equal
-from pyriemann.classification import MDM, FgMDM, KNearestNeighbor, TSclassifier
+from pyriemann.classification import (MDM, FgMDM, KNearestNeighbor,
+                                      TSclassifier, SVC)
+
 from pyriemann.estimation import Covariances
 import pytest
 from pytest import approx
 from sklearn.dummy import DummyClassifier
 
 
-rclf = [MDM, FgMDM, KNearestNeighbor, TSclassifier]
+rclf = [MDM, FgMDM, KNearestNeighbor, TSclassifier, SVC]
 
 
 @pytest.mark.parametrize("classif", rclf)
 class ClassifierTestCase:
     def test_two_classes(self, classif, get_covmats, get_labels):
-        n_classes, n_trials, n_channels = 2, 6, 3
-        covmats = get_covmats(n_trials, n_channels)
-        labels = get_labels(n_trials, n_classes)
+        n_classes, n_matrices, n_channels = 2, 6, 3
+        covmats = get_covmats(n_matrices, n_channels)
+        labels = get_labels(n_matrices, n_classes)
         self.clf_predict(classif, covmats, labels)
         self.clf_fit_independence(classif, covmats, labels)
         if classif is MDM:
```

Comment:
Currently, SVC is not really tested. It must be added in each if-line below:
```suggestion
        if classif is (MDM, SVC):
```